dace/frontend/python/nested_call.py

```diff
@@ -0,0 +1,30 @@
+import dace
+from dace.sdfg import SDFG, SDFGState
+
+class NestedCall():
```

Comment:
When writing the pytorch function replacements, I plan to reuse the existing replacements. To do this, I would propose this `NestedCall` object which allows you to call other replacements while building a replacement

Reply:
It also works with replacements that return stuff, i.e. `result = nest(_func)(....)` should also work.

Reply:
@orausch please add an example use to the documentation, no one will read these comments once the code is merged :)